app/helpers/action_links_helper.rb

```diff
@@ -153,11 +153,8 @@ def display_renew_link_for?(resource)
     resource.can_start_renewal?
   end
 
-  def display_transfer_link_for?(resource)
-    return false if WasteCarriersEngine::FeatureToggle.active?(:block_front_end_logins)
-    return false unless display_registration_links?(resource)
-
-    can?(:transfer_registration, WasteCarriersEngine::Registration)
+  def display_transfer_link_for?(_resource)
```

Comment:
I think this should just be removed

Reply:
removed

Reply:
It's still there

Reply:
looks like my previous commit hasn't been pushed. That code is gone now